scripts/spider_build.sh

```diff
@@ -0,0 +1,5 @@
+if [ "$(basename "$PWD")" = 'script' ]; then cd ..; fi
+
+fvm dart pub global activate spider &&
+  spider build &&
+  fvm dart format .
```

Comment:
To avoid depending on the global Dart version in the project, we can add `spider` to the `dev_dependencies`. Then, for code generation, we can use the command: `fvm flutter pub run spider build`.

Reply:
\+